apps/api/src/app/pps/domain/pps-profile-dto-to-runner-personal-infos.ts

```diff
@@ -21,6 +21,6 @@ export class PPSProfileDTOToRunnerPersonalInfos {
     this.firstname = ppsDto.firstname;
     this.gender = ppsDto.gender;
     this.lastname = ppsDto.lastname;
-    this.eventDate = DateTime.fromISO(ppsDto.event_date);
+    this.eventDate = DateTime.fromJSDate(ppsDto.event_date);
```

Comment:
What is the point of this change? Was the use of `fromISO` correct in my opinion?

Reply:
Before, we used a date string in the dto, but now we use a real date, that's why I needed to change it to `fromJSDate` instead of `fromISO`